core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -691,6 +691,14 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  private[spark] val KERBEROS_FILESYSTEM_RENEWAL_EXCLUDE =
+    ConfigBuilder("spark.kerberos.renewal.exclude.hadoopFileSystems")
+      .doc("The list of Hadoop filesystem URLs for which to exclude from delegation token renewal.")
```

Comment:
This might mislead the users because this configuration bans all file system from that host.

Reply:
This configuration doesn't ban the file system, but excludes it from DT renewal. The user needs to use the file system but YARN cannot renew it.

Reply:
I think I got your point. Update the config doc.

Reply:
Thanks!